win32/src/threadpool_win32.c

```diff
@@ -107,6 +109,16 @@ static void threadpool_dispose(THREADPOOL* threadpool)
     execution_engine_dec_ref(threadpool->execution_engine);
 }
 
+static void threadpool_dispose_work_item(THREADPOOL_WORK_ITEM_CONTEXT_HANDLE threadpool_work_item_context)
+{
+    THREADPOOL_WORK_ITEM_CONTEXT_HANDLE threadpool_work_item_context_ptr = THANDLE_GET_T(THREADPOOL_WORK_ITEM_CONTEXT)(threadpool_work_item_context);

```

Comment:
>THANDLE_GET_T [](http://example.com/codeflow?start=75&length=13)

not needed, the dispose function takes a type `T*` for the parameter, not a `THANDLE(T)`. Can remove this line.

Reply:
Fixed.